assets/quick-order-form.css

```diff
@@ -20,4 +36,95 @@
 .quick-order-form .variant-item__price .price,
 .quick-order-form .variant-item__totals .price {
   font-size: 1.4rem;
-}
\ No newline at end of file
+}
+
+.quick-order-form tr.empty-row {
+  box-shadow: inset 0px 0.1rem 0px 0px rgba(var(--color-foreground), 0.08);
+}
+
+.quick-order-form .quick-order-list__table th {
+  border: 0;
+}
+
+.quick-order-form tr.variant-item:not(.product):hover {
+  background-color: rgba(var(--color-foreground), 0.02);
+}
+
+.quick-order-form tr.variant-item td.variant-item__inner {
+  display: table-cell;
+}
+
+.quick-order-form .quick-order-list__table td {
+  padding-top: 0.4rem;
+  padding-bottom: 0.4rem;
+}
+
+.quick-order-form tr.product td {
+  min-height: calc((var(--inputs-border-width) * 2) + 5rem);
+  padding-top: 1.6rem;
+  padding-bottom: 1.6rem;
+}
+
+.quick-order-form .product-variant__title--wrapper {
+  display: flex;
+}
+
+.quick-order-form tr.product .variant-item__image-container {
+  width: 3.6rem;
+  height: auto;
+  margin-right: 0.8rem;
+  border: 0;
+}
+
+.quick-order-form tr:not(.product) .variant-item__image-container {
+  width: 2.6rem;
+  height: 2.6rem;
+  margin-right: 0.8rem;
+  background-color: transparent;
+  border: 0;
+}
+
+.quick-order-form tr:not(.product) .variant-item__media {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-right: 0.8rem;
+  width: 2.6rem;
+  height: 4rem;
+}
+
+.quick-order-form tr.product .variant-item__media {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-right: 0.8rem;
+  width: 3.6rem;
+  height: auto;
+}
+
+.quick-order-form .field {
+  margin-bottom: 1.8rem;
+}
+
+.quick-order-form .field label {
+  opacity: 0.3;
+}
+
+.search--wrapper {
+  display: flex;
+}
+
+.total-items,
+.seach__filters {
+  white-space: nowrap;
+  line-height: calc(1 + .5 / var(--font-body-scale));
+  font-size: 1.4rem;
+  margin-left: 1.6rem;
+  margin-bottom: 1.8rem;
+  display: flex;
+  align-items: center;
+}
+
+.seach__filters span {
+  margin-left: 0.8rem;
+}
```

Comment:
The loader isnt aligned https://screenshot.click/02-11-6vtd9-qbhu1.mp4

Reply:
Hm 🤔, I don't know why you got this misalignment. I noticed there is a bit of misalignment but not that huge that you see. 

https://screenshot.click/02-08-zlq1t-e2383.mp4